src/renderable/animationsheet.js

```diff
@@ -19,11 +19,12 @@
      * @param {Number} [settings.framewidth] Width of a single frame within the spritesheet
      * @param {Number} [settings.frameheight] Height of a single frame within the spritesheet
```

Comment:
Yeah, those lines have changed since the `Sprite` is responsible for drawing itself relative to its own anchor point now. So the problem is not in `me.Entity.draw`, it's in `me.Sprite.draw`.

I'm almost done, you'll see what I'm doing in a couple of minutes.